pkgs/development/tools/rust/sqlx-cli/default.nix

```
@@ -1,4 +1,4 @@
-{ lib, rustPlatform, fetchFromGitHub, pkg-config, openssl }:
+{ stdenv, lib, rustPlatform, fetchFromGitHub, pkg-config, openssl, darwin }:
```

```suggestion
{ stdenv, lib, rustPlatform, fetchFromGitHub, pkg-config, openssl, SystemConfiguration. CoreFoundation, Security }:
```

and inherit them from top-level.nix.